alerts/elasticsearch-gke/README.md

```diff
@@ -0,0 +1,36 @@
+# Elasticsearch Alerts for GKE
+
+## Yellow cluster status alert
+When the cluster has a "yellow" status, at least one primary shard is missing, and data is missing.
+
+## Red cluster status alert
+When the cluster has a "red" status, at least one primary shard is missing, and data is missing.
+
+## High JVM memory heap usage alert
+When the ratio of heap used to max heap becomes high, then application's performance may start to degrade.
+
+### Creating notification Channels and User Labels
+
+Whether these alert policies are being used as standalones or base templates for a deployment strategy like terraform, one thing that should be utilized is notification channels and user labels.
+
+### User Labels
+
+Supplying user labels could give extra identification information about the firing alert:
+
+i.e.
+
+```json
+    "userLabels": {
+        "datacenter": "central"
```

Comment:
Could we update this to be severity level? i.e. "Severity" : "Warning".

Since the datacenter/location is typically automatically available from the underlying monitored resource. 

Could you please make this edit in the readme's below as well?